packages/auth/src/app.ts

```diff
@@ -195,7 +195,9 @@ export class App {
         path: '/',
         method: HttpMethod.POST
       }),
-      grantInitiationHttpsigMiddleware,
+      this.config.bypassSignatureValidation
```

Comment:
Should the existing bypass be removed?
https://github.com/interledger/rafiki/blob/d808c85c428bfc5bc1193882905ce5fdc8fa5888/packages/auth/src/signature/middleware.ts#L25-L28

Reply:
it is pretty redundant now, yeah.